content/en/docs/Writing policies/variables.md

```diff
@@ -135,193 +168,28 @@ To refer to values from a ConfigMap inside a `rule`, define a `context` inside t
-{{% /alert %}}
-
-Now that the array data is saved in the `allowed-roles` key, here is a sample `ClusterPolicy` containing a single `rule` that blocks a Deployment if the value of the annotation named `role` is not in the allowed list:
-
-```yaml
-apiVersion: kyverno.io/v1
-kind: ClusterPolicy
-metadata:
-  name: cm-array-example
-spec:
-  validationFailureAction: enforce
-  background: false
-  rules:
-  - name: validate-role-annotation
-    context:
-      - name: roles-dictionary
-        configMap:
-          name: roles-dictionary
-          namespace: default
-    match:
-      resources:
-        kinds:
-        - Deployment
-    validate:
-      message: "The role {{ request.object.metadata.annotations.role }} is not in the allowed list of roles: {{ \"roles-dictionary\".data.\"allowed-roles\" }}."
-      deny:
-        conditions:
-        - key: "{{ request.object.metadata.annotations.role }}"
-          operator: NotIn
-          value:  "{{ \"roles-dictionary\".data.\"allowed-roles\" }}"
-```
-
-This rule denies the request for a new Deployment if the annotation `role` is not found in the array we defined in the earlier ConfigMap named `roles-dictionary`. 
-
-{{% alert title="Note" color="info" %}}
-You may also notice that this sample uses variables from both `AdmissionReview` and ConfigMap sources in a single rule. This combination can prove to be very powerful and flexible in crafting useful policies.
-{{% /alert %}}
-
-Once creating this sample `ClusterPolicy`, attempt to create a new Deployment where the annotation `role=super-user` and test the result.
-
-```yaml
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: busybox
-  annotations:
-    role: super-user
-  labels:
-    app: busybox
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: busybox
-  template:
-    metadata:
-      labels:
-        app: busybox
-    spec:
-      containers:
-      - image: busybox:1.28
-        name: busybox
-        command: ["sleep", "9999"]
-```
-
-Submit the manifest and see how Kyverno reacts.
-
-```sh
-kubectl create -f deploy.yaml
-```
-```
-Error from server: error when creating "deploy.yaml": admission webhook "validate.kyverno.svc" denied the request:
-
-resource Deployment/default/busybox was blocked due to the following policies
-
-cm-array-example:
-  validate-role-annotation: 'The role super-user is not in the allowed list of roles: ["cluster-admin", "cluster-operator", "tenant-admin"].'
-```
-
-Changing the `role` annotation to one of the values present in the ConfigMap, for example `tenant-admin`, allows the Deployment resource to be created.
-
-### Nested Lookups
+## Nested Lookups
 
 It is also possible to nest JMESPath expression inside one another when mixing data sourced from a ConfigMap and AdmissionReview, for example. By including one JMESPath expression inside the other, Kyverno will first substitute the inner expression before building the outer one as seen in the below example.
```

Comment:
"to nest JMESPath _expressions_ inside one another..." (this one looks to be my bad)